config.json

```diff
@@ -846,9 +846,11 @@
       "slug": "word-search",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 4,
       "topics": [
-
+        "Matrices",
+        "Searching",
+        "Strings"
```

Comment:
There is probably looping in everything, I've been trying to limit it to where it may be particularly intense.